frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -38,6 +38,9 @@ const CAPACITY_STORAGE_USED_QUERY = 'sum(kubelet_volume_stats_used_bytes)';
 const CAPACITY_STORAGE_TOTAL_DEFAULT_QUERY = 'sum(node_filesystem_avail_bytes)';
 const CAPACITY_STORAGE_USED_DEFAULT_QUERY = 'sum(node_filesystem_avail_bytes) - sum(node_filesystem_free_bytes)';
 
+const CEPH_OSD_UP_QUERY = 'sum(ceph_osd_up)';
```

Comment:
these queries are also used in `storage-overview`. Can we make some common file for them? 

Ideally in the future, we would like to reuse the results as well when switching between tabs..

Reply:
The cluster dashboard and the storage dashboard belong to different components (`kubevirt` vs `storage`). Does it really make sense to tie them together like that?

Reply:
I think it should be okay. @rawagner @mareklibra thoughts?

Reply:
+1 for common file

Reply:
OK, sounds good. What would be a good place for it?

Reply:
hmmm, it may be too much to create another folder for that in `web-ui`. I would just expose it from `kubevirt-web-ui-components`. Maybe add it to `constants/storage.js` and create an object

```js
const STORAGE_PROMETHEUS_QUERIES = {
 CEPH_OSD_UP_QUERY: '...'
  ...
}
```

again @rawagner @mareklibra thoughts?

Reply:
I agree, `web-ui-components` is definitely the place to move these to. `constants/prometheus.js` would be better IMO.

Reply:
Done